db/Models/index.js

```diff
@@ -87,13 +87,17 @@ const Tutor = db.define("Tutor", {
   }
 });
 
-Parent.hasMany(Student);
-Student.hasMany(Parent);
-Parent.belongsToMany(Student, { through: "ParentStudent" });
-Student.belongsToMany(Parent, { through: "ParentStudent" });
```

Comment:
Strings don't work, and I am not sure why. But we use the `sequelize` Model object instead.